tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/RabbitMQIntegration.cs

```diff
@@ -68,6 +72,21 @@ internal static Scope CreateScope(Tracer tracer, out RabbitMQTags tags, string c
             return scope;
         }
 
+        private static string GetOperationName(Tracer tracer, string spanKind)
```

Comment:
I agree it would be nice to test this as unit tests. Have been trying today to test it but Moq is too fussy about only being able to mock virtual things, and I'm not a fan of changing code for tests. I would consider the integration tests to be enough, WDYT?

Reply:
Scratch that, managed to get it working, see 9321b4bc0dd9e0563c0a16c4e43467bb8af3d2c2.

Still, I believe it doesn't add much, but it's now there so I'm keeping it.

Reply:
I see, I followed a similar approach:

https://github.com/DataDog/dd-trace-dotnet/pull/4386/files#diff-e3808998541c20943d8f574d1370aacc63bec315cea2280927d8b73f2f7efd2f

Just wanted to make sure!